### DISCUSSION TITLE
Is there a way to set a seed for cursor strings for the sake of testing?

### DISCUSSION THREAD

**Discussion by zromick** [uriyyo/fastapi-pagination#1197] [Q&A] [answered]
I'd like to run some tests in Pytest to ensure fast testing for cursor. Is there a way to configure my testing library to get the same strings back for cursor results every time? Or perhaps an alternative way to test? Thanks!

**Comment by uriyyo** [comment, marked as answer]
Hi @zromick,

You can try to mock `paginate` function that you are using. 

Here is example - https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.return_value
